### PR TITLE
Updated the typos in TF doc

### PR DESCRIPTION
**Pull request by sushreebarsa**
There were some typos in the documentation which are addressed in this PR.

# Description

> :memo: Please include a summary of the change. 
>  
> * Please also include relevant motivation and context.  
> * List any dependencies that are required for this change.  

## Type of change

For a new feature or function, please create an issue first to discuss it
with us before submitting a pull request.

Note: Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] TensorFlow 2 migration
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] A new research paper code implementation
- [ ] Other (Specify)

## Tests

> :memo: Please describe the tests that you ran to verify your changes.
>  
> * Provide instructions so we can reproduce.  
> * Please also list any relevant details for your test configuration.  

**Test Configuration**:

## Checklist

- [ ] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [ ] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [ ] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [ ] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: docs/nlp/fine_tune_bert.ipynb [hunk 1]
```diff
@@ -73,9 +73,9 @@
...
+        "You can also find the pre-trained BERT model used in this tutorial on [TensorFlow Hub (TF Hub)](https://tensorflow.org/hub). For concrete examples of how to use the models from TF Hub, refer to the [Solve Glue tasks using the BERT](https://www.tensorflow.org/text/tutorials/bert_glue) tutorial. If you're just trying to fine-tune a model, the TF Hub tutorial is a good starting point.\n",
         "\n",
-        "On the other hand, if you're interested in deeper customization, follow this tutorial. It shows how to do a lot of things manually, so you can learn how you can customize the workflow from data preprocessing to training, exporting and saving the model."
+        "On the other hand, if you're interested in deeper customization, follow this tutorial. It shows how to do a lot of things manually, so you can learn how you can customize the workflow from data preprocessing to training, exporting ,and saving the model."
       ]
```
Comment (saberkun) [inline docs/nlp/fine_tune_bert.ipynb:79]: revert? or exporting, and saving

**Review by saberkun — CHANGES_REQUESTED** [review]

File: docs/nlp/fine_tune_bert.ipynb [hunk 2]
```diff
@@ -1189,7 +1189,7 @@
         "id": "qnxysGUfIgFQ"
       },
       "source": [
-        "Create an instance of this export-model and save it:"
+        "Create an instance of this export model and save it:"
```
Comment (luotigerlsx) [inline docs/nlp/fine_tune_bert.ipynb:1192]: Keeping dash here makes more sense I think. Or we can make it "exported model".

File: docs/nlp/fine_tune_bert.ipynb (hunk 1 above)
Comment (sushreebarsa) [reply docs/nlp/fine_tune_bert.ipynb:79]: Sounds good! Thank you!

File: docs/nlp/fine_tune_bert.ipynb (hunk 2 above)
Comment (sushreebarsa) [reply docs/nlp/fine_tune_bert.ipynb]: Sure! LGTM. Thank you!

**Comment by gbaned** [comment]
Hi @luotigerlsx Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @luotigerlsx Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @luotigerlsx Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @Chen-Zhe Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @Chen-Zhe Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @Chen-Zhe Can you please review this PR ? Thank you!

**Comment by laxmareddyp** [comment]
Closing this PR as these changes submitted through [internal CL](https://github.com/tensorflow/models/commit/8e9f80a27e4abd2ed889701cd34d42ef2a6e9373)

**Pull request closed without merging**